### PR TITLE
doc(README) : Migrate to adoc format and doing few improvements

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
Getting Started with

## What
- Migrate it for doc format which is the team doc definition,
- Add some steps and small improvements

## Why
- Make easier the first contact
- Follow the standards definitions

## Verification Steps
Check: 
https://github.com/aerogear/aerogear-unifiedpush-server/blob/bce97ac13a1cb2972d97faf9297c84dd9f559477/README.adoc

## Checklist:

- [X] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+----
+docker-compose -f docker-compose-v2.1.yaml up
+----
+
+IMPORTANT: There is a bug on docker-compose for Mac with does not expose internal docker IP. To fix it you will need following the steps in <<Solving bug in Mac OS to use Docker-Compose>>
+
+==== Solving bug in Mac OS to use Docker-Compose
+
+. Expose the Keycloak port in the link:./docker-compose/docker-compose-v2.1.yaml[docker-compose-v2.1]
++
```
Comment (camilamacedo86) [inline README.adoc]: @danielpassos this step is not 100% clear for me. How to  Expose the Keycloak port? Is not it done by default already? What exactly we need to do here? Could you help by describing it?

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+| Documentation:  | https://docs.aerogear.org
+| Issue tracker:  | https://issues.jboss.org/browse/AEROGEAR
+| Mailing lists:  | https://groups.google.com/forum/#!forum/aerogear[aerogear-dev]
+|===
+
+== Prerequisites
+
+|===
+| https://maven.apache.org/install.html[Maven]
+| https://docs.docker.com/install/[Docker]
```
Comment (camilamacedo86) [inline README.adoc]: @danielpassos @secondsun is it supporting any version? What are the minimum versions required?

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+| `CUSTOM_AEROGEAR_FCM_PUSH_HOST` | Custom host for sending Google Firebase push notifications. Can be used for testing.
+| `UPS_REALM_NAME` | Override Keycloak Realm.
+| `KEYCLOAK_SERVICE_HOST` | Override Keycloak authentication redirect.
+| `AEROGEAR_METRICS_STORAGE_DAYS` | Override the number of days the metrics are stored (default is 30 days).
+| `ARTEMIS_URL` | URL For AMQP Server.
+| `ARTEMIS_PORT` |  PORT For AMQP Server.
+| `ARTEMIS_PASSWORD` |  Password for AMQP server.
+| `ARTEMIS_USERNAME` |  Username for AMQP server.
+|===
+
```
Comment (camilamacedo86) [inline README.adoc]: @danielpassos it is missing the steps over Development. 
E.g ( local env, how to test any change made in the code locally, debug and hot deploy steps)

File: README.adoc (hunk 2 above)
Comment (danielpassos) [reply README.adoc]: Nopz!

File: README.adoc [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+==== Using Docker-Compose
+
+For your convenience, we do have an easy way of launch different configurations of the UPS, using our link:./docker-compose[Docker compose files]
+
+By the following command you can startup this project locally using Docker-Compose.
+
+[source,shell]
+----
+docker-compose -f docker-compose-v2.1.yaml up
```
Comment (danielpassos) [inline README.adoc]: Docker image was renamed to the default name docker-compose.yaml

File: README.adoc (hunk 1 above)
Comment (danielpassos) [reply README.adoc]: It's not needed anymore. I have sent a PR [exposing the port](https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/docker-compose/docker-compose.yaml#L51) already.

Comment (danielpassos) [reply README.adoc]: Actually I think we should avoid the docker-compose details here and link this section to the docker-compose folder README

**Comment by camilamacedo86** [comment]
Hi @danielpassos the changes requested are done. Could you please check if it is ok now?

File: README.adoc (hunk 3 above)
Comment (camilamacedo86) [reply README.adoc]: Let's move forward and add it in a second moment.

File: docker-compose/README.adoc [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+[source,shell]
+----
+docker-compose -f docker-compose.yaml up
+----
+
+IMPORTANT: There is a bug on docker-compose for Mac with does not expose internal docker IP. To fix it you will need following the steps in <<Solving bug in Mac OS to use Docker-Compose>>
+
+== Solving bug in Mac OS to use Docker-Compose
+
+. Expose the Keycloak port in the link:docker-compose.yaml[docker-compose.yaml]
```
Comment (danielpassos) [inline docker-compose/README.adoc]: This step is not needed anymore and should be excluded. We already have added this in the commited `docker-compose.yaml` See: https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/docker-compose/docker-compose.yaml#L50-L51

Comment (camilamacedo86) [reply docker-compose/README.adoc]: Now ti makes more sense. Really tks :-)
Done 👍

**Pull request merged**